src/main/java/no/fintlabs/kafka/common/ListenerConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package no.fintlabs.kafka.common;
+
+import org.springframework.kafka.listener.CommonErrorHandler;
+
+public interface ListenerConfiguration {
```

Comment:
Grunnen til at ListenerConfiguration er en interface i stedet for at field-ene arves av de spesialiserte configuration-ene, er at det er klønete å skjule/sette fast verdi på enkelte felter med kombinasjon av arv og builder-mønster. For eksempel med at resettingOffsetOnAssignment er fastsatt til true i EntityConsumerConfiguration. Det gjør det også lettere å spesialtilpasse de forskjellige konfigurasjonene i fremtiden.